lms/templates/open_ended_problems/combined_notifications.html

```diff
@@ -1,51 +0,0 @@
-<%! from django.utils.translation import ugettext as _ %>
-<%inherit file="/main.html" />
-<%block name="bodyclass">${course.css_class}</%block>
-<%namespace name='static' file='/static_content.html'/>
-
-<%block name="headextra">
-<%static:css group='style-course-vendor'/>
-<%static:css group='style-course'/>
-</%block>
-
-<%block name="pagetitle">${_("{course_number} Combined Notifications").format(course_number=course.display_number_with_default)}</%block>
-
-<%include file="/courseware/course_navigation.html" args="active_page='open_ended'" />
-
-
-<section class="container">
-    <div class="combined-notifications" data-ajax_url="${ajax_url}">
-        <div class="error-container">${error_text}</div>
-
-        <h1>${_("Open Ended Console")}</h1>
-        <h2>${_("Instructions")}</h2>
-        <p>${_("Here are items that could potentially need your attention.")}</p>
-        % if success:
-            % if len(notification_list) == 0:
-            <div class="message-container">
-              ${_("No items require attention at the moment.")}
-            </div>
-            %else:
-            <div class="notification-container">
-                %for notification in notification_list:
-                 % if notification['alert']:
-                    <div class="notification alert">
-                % else:
-                    <div class="notification">
-                % endif
-                    <a href="${notification['url']}" class="notification-link">
-                        <div class="notification-title">${notification['name']}</div>
-                        %if notification['alert']:
-                        <p class="alert-message"><img src="${static.url('images/grading_notification.png')}" /> ${notification['alert_message']}</p>
```

Comment:
This image is removed here: https://github.com/edx/edx-platform/commit/b18c9fc3b1bfa7d0e0034ecbea99367c7b42bc1a